src/pages/Facility/locations/LocationNavbar.tsx

```
@@ -50,9 +50,17 @@
     }),
     enabled: true,
   });
-
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)